src/Input/Textarea/Textarea.tsx

```diff
@@ -145,14 +128,6 @@ interface Props extends React.ComponentPropsWithoutRef<typeof TextareaInput> {
   forwardedRef?: React.RefObject<HTMLTextAreaElement>;
```

Comment:
Sorry, I cannot leave a comment on `onBlur`. Could you help to add `(e: React.FocueEvent<HTMLTextAreaElement>)` for it?

Reply:
Updated!